jbmc/regression/jbmc-strings/append_obj_int_tostring/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE symex-driven-lazy-loading-expected-failure
+Main.class
+--lazy-methods --stop-on-fail --refine-strings --string-max-length 10 --unwind 10
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+--
```

Comment:
Since this is a _regression_ test: when was this an issue, could the commit message maybe say something to that effect?

Reply:
Would require a bisect between now and January.